io/net/multicast.py

```diff
@@ -79,6 +79,11 @@ def test_multicast(self):
         if process.system(cmd, shell=True, verbose=True,
                           ignore_status=True) != 0:
             self.fail("unable to set all mulicast option to test interface")
+        peer_route_add = "ip route add 224.0.0.0/4 dev %s" % self.peerif
```

Comment:
I would rather reuse the variable `msg` instead of introducing another temporary variable. Except that, looks fine to me.

Reply:
Why not keep that variable which tells the action being performed?

Reply:
It is aboutuniformity